sale_project_timesheet_by_seniority/tests/test_sale_project_timesheet_by_seniority.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo.exceptions import UserError
+from odoo.tests.common import SavepointCase
+
+
+class TestSaleProjectTimesheetBySeniority(SavepointCase):
+    """  """
```

Comment:
What's use of that?

Reply:
To remind me to write something, did not work, removed.